src/logtab.cpp

```diff
@@ -612,12 +618,49 @@ void LogTab::ChangePrevIndex()
     }
 }
 
+void NormalizeLogText(QString& logText)
+{
+    logText.replace(QRegularExpression("id: (-)?\\d+"), "id: N");
```

Comment:
Make this a list of key-value pairs (or dictionary) and just iterate through that.
Would look a bit cleaner.